app_dart/lib/src/service/scheduler.dart

```diff
@@ -104,6 +104,11 @@ class Scheduler {
   }
 
   Future<void> _addCommit(Commit commit) async {
+    if (!Config.schedulerSupportedRepos.contains(commit.repository)) {
+      log.debug('Skipping ${commit.id} as repo is not supported');
```

Comment:
Nit: use $repo instead.

Reply:
`commit.id` includes `$repository/$branch/$sha` so that's why I favored it

Reply:
makes sense, LGTM